AppCenterAuth/AppCenterAuth/Internals/Models/MSB2CAuthority.m

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSB2CAuthority.h"
+#import "../Util/MSAuthConstants.h"
+
+@implementation MSB2CAuthority
+
+static NSString *const kMSType = @"type";
```

Comment:
We can move "type" and "authority_url" constants to the parent class, right?

Reply:
No its not. Removed it.